gazelle/python/testdata/remove_invalid_library/BUILD.out

```diff
@@ -0,0 +1,6 @@
+load("@rules_python//python:defs.bzl", "py_library")
+
+py_library(
+    name = "deps_with_no_srcs_library",
+    deps = ["//:remove_invalid_library"],
```

Comment:
nit: maybe having a different dep, that is not the one that gets deleted would be slightly less confusing.

Would having `@pypi//foo` and `@pypi//bar` work here?

Reply:
added